src/main/java/games/strategy/engine/framework/ui/GameChooserModel.java

```diff
@@ -89,7 +89,7 @@ public GameChooserEntry get(final int i) {
       } catch (final InterruptedException e) {
         Thread.currentThread().interrupt();
       } catch (final ExecutionException e) {
-        ClientLogger.logError(e.getCause());
+        ClientLogger.logError("Failed to parse a map", e);
```

Comment:
`ExecutionException` contains no useful information; it's simply an adapter to tunnel exceptions thrown by tasks through the `Executor` API.  We've been logging the cause directly to avoid the addition of 10-20 useless stack frames from the overall stack trace.

Reply:
Updated and added a comment: e886bd4f9